src/test/java/ru/skypro/homework/controller/UserControllerTest.java

```diff
@@ -90,7 +91,7 @@ public void getUserTest() throws Exception {
                 .apply(springSecurity())
                 .build();
 
-        UserDetails userDetails = userDetailsService.loadUserByUsername("petrov@gmail.com");
+        UserAuth userDetails = (UserAuth) userDetailsService.loadUserByUsername("petrov@gmail.com");
```

Comment:
```suggestion
        UserDetails userDetails = userDetailsService.loadUserByUsername("petrov@gmail.com");
```
каст никакой не нужен